packages/jest-cli/src/__tests__/watch.test.js

```diff
@@ -86,6 +86,11 @@ jest.doMock(
   {virtual: true},
 );
 
+const regularUpdateGlobalConfig = require('../lib/update_global_config')
```

Comment:
You can `import` it to avoid the `.default` call

Reply:
Yes, but as I'm mocking it right the next line, it seemed more interesting to keep it close by?

Reply:
`jest.doMock` should be called prior to require call